client/js/player/scoreboard.js

```diff
@@ -1,50 +1,51 @@
 define(function(require) {
 
+var Protocol = require("core/protocol");
+
 return function Scoreboard(controls, conn, container) {
 	var self = this;
 	var scoreboard = document.getElementById("scoreboard");
 	var scores = {};
 	var dirty = false;
 
-	conn.on('scoreboard-add', handleAdd);
-	conn.on('scoreboard-set', handleSet);
-	conn.on('scoreboard-remove', handleRemove);
+	conn.on(Protocol.MSG_SCOREBOARD_ADD, function(result) {
```

Comment:
Should not call it result probably, maybe "msg" might be clearer.

At least it's consistent though!